src/containers/add-documents/AddDocuments.constants.ts

```diff
@@ -7,5 +7,7 @@ export const validationData: AddDocuments = {
   fileSizeError: 'common.fileSizeError',
   allFilesSizeError: 'common.allFilesSizeError',
   typeError: 'common.typeError',
-  maxQuantityFiles: 7
+  maxQuantityFiles: 7,
```

Comment:
1. Seems like we limit number of files uploaded in one attempt, but check is not used inside validations
2. I noticed, that if I upload multiple files, they upload in random order (I believe it depends on file size or filename)
3. It would be great user experience if we sort these files in some way

Maybe it can be moved into new issue or fixed here

Reply:
```maxQuantityFiles``` is used in hook ```use-upload``` checks, it is needed for slicing all files, which are user select through the input, and than ```filesValidaion``` checks files. So it's basically not possible to get more than 7 files

about sorting, i think i can look what can we do here

Reply:
In my opinion it is bad user experience, because if user wants to upload 20 files (for example his or her folder with resources) he actually uploads only 7 files and it is confusing and not user-friendly. I think we should inform user about it in validation

Reply:
added error for max quantity, as for sorting it should be backend task to add sort by creation when fetching attachments

Reply:
It shows error "Maximum allowed to add 7 photos at once" for the first time, but second time when i attempt to load more than 7 files at once i don`t get an error and files are uploaded. Maybe something is not reset properly

Reply:
will fix it in another issue